fixes #29

Web-Dev-Rich opened the pull request:
Amends code snippet to show where it goes in server.js more clearly. Uses the actual code from server.js.

Commit:
fixes #29

johnstonbl01 commented:
Hi there. Rather than have a single `.` to imply missing code, can you please update to have `...` imply code -- this is the convention used in other parts of the tutorial. Let me know if you have any questions.

Commit:
Implied missing code character change fixes #29

Change implied missing code character from '.' on three separate lines to a single '...'

johnstonbl01 commented:
Thanks so much for the contribution!
